modules/utxo-lib/test/networks.ts

```diff
@@ -133,7 +133,10 @@ describe('networks', function () {
       it('has expected properties', function () {
         assert.strictEqual(typeof network, 'object');
         assert.strictEqual(typeof network.messagePrefix, 'string');
-        assert.strictEqual(typeof (network as any).bech32, bech32Coins(network) ? 'string' : 'undefined');
+        assert.strictEqual(typeof (network as any).bech32,  'string');
+        if (!bech32Coins(network)) {
+          assert.strictEqual((network as any).bech32.length, 0);
+        }
```

Comment:
IMO `undefined` is better than zero-length string here

Reply:
No disagreement, but that's not what the upstream type has.

Reply:
changed it so we do not use the upstream type anymore: https://github.com/BitGo/BitGoJS/pull/2054/commits/6f553c087ff01d433b1cd8498dde583dc47960a6